app/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tidepool-uploader",
   "productName": "tidepool-uploader",
-  "version": "2.30.1",
+  "version": "2.30.0-i-sens-hid.1",
```

Comment:
Now that I've switched to GitHub flow, I'l be removing this version tag after Eric completes QA, and then merge it into `master` still as `2.30.1`. I guess I should've made it `.1` instead of .0` here, but it's the suffix that matters for QA. I will only rev the version number to `2.31.0` when I actually push a new version to prod.